src/renderer/components/shared/SharedUtils.js

```diff
@@ -1,21 +1,33 @@
 export const META_CONCEPTS = new Set(['entity', 'relation', 'attribute', 'role']);
 
+export const interfaceTypes = {
```

Comment:
what's interfaceTypes? can we find a more descriptive name?

Reply:
I'm not sure how else to put it. perhaps the properties on the object clarify what's meant by an interface?

Reply:
ok I understand what's this now,
what do you think of something like:
`edgeType: VISUALISER_EDGE | SCHEMA_DESIGNER_EDGE`?

Reply:
I think what determines the type of an edge (or a node) is, in fact, the type of the interface they are displayed on. The reason why this may feel unfamiliar could be because `interface` doesn't currently exist in our terminology. We currently have the term `schema designer` and `visualiser`, and `interface` is meant to be used as the term that describes what both the schema designer and visualiser are.